crates/fluvio-spu-schema/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio-spu-schema"
-version = "0.18.1"
+version = "0.18.2"
```

Comment:
Should be 0.19.0. I think is ok another PR for that.

Reply:
the merge group failed, so I can push an update